README.rst

```diff
@@ -17,7 +17,7 @@ composer.json:
 
     {
         "require": {
-            "guzzlehttp/oauth-subscriber": "0.5.*"
+            "guzzlehttp/oauth-subscriber": ^0.5"
```

Comment:
Missing double quote.

Reply:
sorry, I'm actually coding something else while doing this.